package.json

```diff
@@ -31,28 +31,26 @@
     "build:esm": "tsc -p tsconfig.esm.json && node ./scripts/webpack-esm.js",
     "build:storybook": "build-storybook -o ./docs/storybook/",
     "storybook": "start-storybook",
-    "test": "rm -rf ./unit_test && tsc -p tsconfig.test.json && karma start --coverage",
+    "test": "rm -rf ./unit_test && tsc -p tsconfig.test.json && web-test-runner",
```

Comment:
How will it be if we use --coverage option?
You set coverage: true on config file, so it won't be needed??

Reply:
Yes. 
The coverage in config is whether to analyze code coverage
https://modern-web.dev/docs/test-runner/cli-and-configuration/#cli-flags

Reply:
Got it!